### PR TITLE
chore(refactor): Refactor test command to use test task

### PR DESCRIPTION
**Pull request by Brocco**

### CODE REVIEW

File: addon/ng2/tasks/test.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/* jshint node: true */
+'use strict';
+
+var Promise = require('ember-cli/lib/ext/promise');
+var Task = require('ember-cli/lib/models/task');
+var exec = Promise.denodeify(require('child_process').exec);
```
Comment (gkalpak) [inline addon/ng2/tasks/test.js]: Is this needed ?

Comment (filipesilva) [inline addon/ng2/tasks/test.js]: Doesn't seem to be, no.

Comment (Brocco) [inline addon/ng2/tasks/test.js]: Agreed, fixing and re-pushing momentarily

**Comment by filipesilva** [comment]
lgtm

**Pull request closed without merging**